Vagrantfile

```diff
@@ -88,6 +90,7 @@ Vagrant.configure(2) do |config|
       config.vm.box = 'elastic/debian-9-x86_64'
       deb_common config, box
     end
+    deb_docker config
```

Comment:
This should be moved before `end` in line 91; having it here causes `deb_docker` to be invoked for every machine.